examples/ili9341_shield_simpletest.py

```diff
@@ -36,7 +36,7 @@
 # Draw a smaller inner rectangle
 inner_bitmap = displayio.Bitmap(280, 200, 1)
 inner_palette = displayio.Palette(1)
-inner_palette[0] = 0x65328F # Blinka Purple
+inner_palette[0] = 0xAA0088 # Purple
```

Comment:
Are you sure this is not another case of the RGB/BGR mixup?

Reply:
Yes. Red is red, blue is blue. It's more of a RGB888->RGB565 issue.